current-v4/nocturne.css

```
@@ -0,0 +1,800 @@
+  --composer-active-scale: 1;
+  --composer-send-scale: 1.08;
+  --composer-bounce-distance: 8px;
+  --composer-bounce-overshoot: -3px;
+  --composer-active-ease: cubic-bezier(0.34, 1.56, 0.64, 1);
+  --composer-send-ease: cubic-bezier(0.34, 1.56, 0.64, 1);
+  --composer-active-duration: 320ms;
+  --composer-send-duration: 520ms;
+}
+
+* {
+  font-family: 'Lexend', 'Segoe UI Emoji', sans-serif !important;
```

Stylelint (17.7.0)

[error] Expected no quotes around "Lexend" (font-family-name-quotes)

(font-family-name-quotes)